src/lat/compose-lattice-pruned.cc

```diff
@@ -159,6 +159,9 @@ class PrunedCompactLatticeComposer {
     int32 lat_state;
     int32 lm_state;
 
+    // the distance between the state from the start state
+    int32 depth;
```

Comment:
it might be more accurate to describe this as:
```
the number of arcs on the path from the start state to this state, in the composed
lattice, by which this state was first reached.
```